sdks/builds/Makefile

```diff
@@ -6,7 +6,8 @@ RELEASE=$(if $(filter $(CONFIGURATION),release),1)
 
 CCACHE:=$(shell which ccache)
 
-ifneq ($(shell uname),"Darwin")
```

Comment:
What's the difference? How is it breaking?

Reply:
The original line evals to true on osx, breaking the ios sdk lane. Couldn't figure out why that happens, the new version seems to work in both cases.